src/components/TimeRelated.tsx

```diff
@@ -1,33 +1,33 @@
 import React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
 import { WeatherType, WeatherIcon } from '@components/WeatherIcon';
-import { eachHourOfInterval, set, getHours, getDate } from 'date-fns';
+import { getHours } from 'date-fns';
 
-const NowDate = new Date();
+interface IProps {
+    time: string;
+    weatherType: WeatherType;
+    temperature: number;
+}
 
-const ArrayOfHours = eachHourOfInterval({
-    start: NowDate,
-    end: set(NowDate, { date: getDate(NowDate) + 1 }),
-});
+const currentDate = new Date();
 
-const Hours = new Array(ArrayOfHours.length);
+function ISO8601ToHours(inputStr: string) {
```

Comment:
This func gives an error
![image](https://user-images.githubusercontent.com/52620425/224545577-90176d8f-bf72-40ed-b754-03406265045c.png)
Again we are using arrow functions: `const ISO.. = () => {}`
Also i don't understand what are you doing here, it is better to get date-fns's Date object from this string and then get hours of it